dlclibrary/dlcmodelzoo/modelzoo_download.py

```diff
@@ -62,41 +66,58 @@ def download_huggingface_model(modelname, target_dir=".", remove_hf_folder=True)
         Directory where to store the model weights and pose_cfg.yaml file
     remove_hf_folder : bool, default True
         Whether to remove the directory structure provided by HuggingFace after downloading and decompressing data into DeepLabCut format.
+    rename_mapping : dict, default None
+        Dictionary to rename the downloaded file. If None, the original filename is used.
     """
     from huggingface_hub import hf_hub_download
```

Comment:
``` suggestion

import tarfile

from huggingface_hub import hf_hub_download
from pathlib import Path
from ruamel.yaml.comments import CommentedBase
```
isort

Reply:
Is there a reason why imports are not in the header?) let's mb put inputs in the header since the main function that you call from this module is exactly `download_huggingface_model`. So you will import these libs any way:) O